app/src/main/res/layout/topic_practice_header_view.xml

```diff
@@ -21,18 +23,20 @@
       android:text="@string/topic_practice_master_these_skills"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="20sp"
+      android:layout_marginEnd="88dp"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"/>
+      app:layout_constraintTop_toTopOf="parent" />
+
     <TextView
       android:id="@+id/skills_description_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginTop="8dp"
+      android:layout_marginTop="4dp"
       android:fontFamily="sans-serif"
       android:text="@string/topic_practice_skills_description"
       android:textColor="@color/oppiaSecondaryText"
       android:textSize="16sp"
       app:layout_constraintStart_toStartOf="@+id/master_skills_text_view"
-      app:layout_constraintTop_toBottomOf="@+id/master_skills_text_view"/>
+      app:layout_constraintTop_toBottomOf="@+id/master_skills_text_view" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done